cfme/tests/storage/test_object_store_object.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+import pytest
+import random
+
+from cfme.cloud.provider.openstack import OpenStackProvider
+from cfme.storage.object_store_object import ObjectStoreObjectCollection
+from cfme.utils import testgen, version
+
+
+pytestmark = pytest.mark.usefixtures("setup_provider")
+
+
+pytest_generate_tests = testgen.generate([OpenStackProvider], scope="module")
+
+
+@pytest.yield_fixture(scope="module")
+def objects(appliance, provider):
+    collection = ObjectStoreObjectCollection(appliance=appliance)
+    objects = collection.all(provider)
+    yield objects
+
+
+@pytest.mark.uncollectif(lambda: version.current_version() < '5.8')
+@pytest.mark.tier(3)
+def test_add_remove_tag(objects):
```

Comment:
**REQURIED**
Bad naming. Please be more specific, where the tag added and removed.